code/api/src/setup/models.js

```diff
@@ -8,7 +8,9 @@ const models = {
   User: databaseConnection.import('../modules/user/model'),
   Product: databaseConnection.import('../modules/product/model'),
   Crate: databaseConnection.import('../modules/crate/model'),
-  Subscription: databaseConnection.import('../modules/subscription/model')
+  Subscription: databaseConnection.import('../modules/subscription/model'),
+  Shipment: databaseConnection.import('../modules/shipments/model')
```

Comment:
We also needed to add this line to the models file which tells sequelize that the model file is a model

Reply:
Thanks for clarifying that additional line!

Reply:
I trust your judgement!